app/api/__init__.py

```diff
@@ -41,19 +41,22 @@
 
 # events
 api.route(EventList, 'event_list', '/events')
-api.route(EventDetail, 'event_detail', '/events/<int:id>', '/tickets/<int:ticket_id>/event',
+api.route(EventDetail, 'event_detail', '/events/<int:id>', '/events/<identifier>', '/tickets/<int:ticket_id>/event',
           '/microlocations/<int:microlocation_id>/event', '/social_links/<int:social_link_id>/event',
           '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event',
           '/call_for_papers/<int:call_for_paper_id>/event', '/session_types/<int:session_type_id>/event',
           '/event_copyright/<int:copyright_id>/event')
 api.route(EventRelationship, 'event_ticket', '/events/<int:id>/relationships/ticket')
+
```

Comment:
@LuD1161 I see all relationships here seem to be still using only `<int:id>` .. We need identifier support there too. The FE will primarily be using the identifier as the id for querying.